### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by lasse2507a** [sw08-06/model_development#20]
- Fix GPU check
- Shuffle data in model.fit()
- Amount of epochs?
- Batch size?
- Learning rate?
- Loss function and metrics
- Callbacks
